deepdrivemd/data/stream/aggregator_reader.py

```diff
@@ -27,6 +69,10 @@ def next(self, ARW):
 
 
 class scalar_StreamVariable(StreamVariable):
```

Comment:
A small thing, but can class names follow the pep8 style (CamelCase)? Maybe this one could be `StreamScalarVariable` and the CM one could be `StreamCMVariable` or even `StreamContactMapVariable`.

I need to update the DeepDriveMD_API class for this as well. I've been busy and haven't had a time to read through all your changes, but it's looking really good!

Reply:
Done